app/lib/primer/underline_nav_helper.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Primer
+  # Helper to share tab validation logic between components.
+  # The component will raise an error if there are 0 or 2+ selected tabs.
+  module UnderlineNavHelper
```

Comment:
I'm not sure how useful this is anymore, since `UnderlinePanel` calls `UnderlineNav`, all of those methods sould be called only on `UnderlineNav`.
about the constants, we can define them in `UnderlineNav` and make `UnderlinePanel` copy them from the other class